data/json/monstergroups/lab.json

```diff
@@ -162,7 +164,8 @@
       { "monster": "mon_zombie_armored", "freq": 1, "cost_multiplier": 5 },
       { "monster": "mon_zombie_electric", "freq": 10, "cost_multiplier": 3 },
       { "monster": "mon_zombie_necro", "freq": 1, "cost_multiplier": 15 },
-      { "monster": "mon_zombie_brute_shocker", "freq": 1, "cost_multiplier": 5 }
+      { "monster": "mon_zombie_brute_shocker", "freq": 1, "cost_multiplier": 5 },
+      {"monster": "mon_mole_cricket", "freq": 20, "cost_multiplier": 3}
```

Comment:
Same, especially at the same frequency as basic zombies?

Reply:
"GROUP_MICROLAB" doesn't have any moles, so it probably shouldn't have any mole crickets.

Or I can lower freq to 2, similar to talons.